v2.1/performance-best-practices-overview.md

```diff
@@ -48,6 +48,8 @@ When a table is created, all columns are stored as a single column family. This
 
 [Interleaving tables](interleave-in-parent.html) improves query performance by optimizing the key-value structure of closely related tables, attempting to keep data on the same key-value range if it's likely to be read and written together. This is particularly helpful if the tables are frequently joined on the columns that consist of the interleaving relationship.
 
+However, the above is only true for tables where all operations (i.e., [`SELECT`](selection-queries.html), [`INSERT`](insert.html), [`UPDATE`](update.html), and [`DELETE`](delete.html)) are performed on a single value shared between both tables.  After interleaving, operations that span multiple values, or that do not specify the interleaved parent ID, will be significantly slower than they were prior to interleaving the tables.
```

Comment:
I would write `(e.g., SELECT or INSERT)` instead of trying to list all of them. As it is now you're forgetting half a dozen already.

Reply:
Fixing now, thanks.

Follow-up question: Does this information also apply to the following versions?

- 2.0.*
- 2.2

If so I'll add this information there as well.

Reply:
Ok, updated this PR so this info is added to the 2.1 and 2.2 docs.  Raphael, if you confirm it applies to 2.0, I'll add it there as well.

Reply:
> What does the word "significantly" mean? Significant according to which standard? Compared to what?

![screen shot 2019-01-24 at 11 55 46 am](https://user-images.githubusercontent.com/1312291/51694650-46eae000-1fcf-11e9-993e-3a6282c8a97a.png)

😂

Thanks for this additional feedback, Raphael.

I have attempted another revision that includes this information, rewritten slightly for clarity.  Please let me know what you think.